content/en/opentelemetry/ingestion_sampling_with_opentelemetry.md

```diff
@@ -17,83 +17,101 @@ further_reading:
 
-{{< img src="/opentelemetry/guide/ingestion_otel/otel_head_tail_based_sampling.png" alt="Otel APM Metrics computation" style="width:100%;" >}}
+- **Head-based sampling** in the OpenTelemetry SDKs
+- **Tail-based sampling** in the OpenTelemetry Collector
+- **Probabilistic sampling** in the Datadog Agent
+
+### Head-based sampling
 
-### SDK-level sampling
+At the SDK level, you can implement _head-based sampling_. This is when the sampling decision is made at the beginning of the trace. This type of sampling is particularly useful for high-throughput applications, when you have a clear understanding of which traces are most important to ingest and want to make sampling decisions early in the tracing process.
 
-At the SDK level, you can implement _head-based sampling_, which is when the sampling decision is made at the beginning of the trace. This type of sampling is particularly useful for high throughput applications, for which you know that you do not need visibility over 100% of the traffic to monitor the application health. It can also help control the overhead introduced by OpenTelemetry.
+#### Configuring
 
-[TraceIdRatioBased][5] and [ParentBased][6] are the SDK's built-in samplers that allow you to implement deterministic head-based sampling based on the `trace_id` at the SDK level.
+To configure head-based sampling, use the [TraceIdRatioBased][5] or [ParentBased][6] samplers provided by the OpenTelemetry SDKs. These allow you to implement deterministic head-based sampling based on the `trace_id` at the SDK level.
 
-With head-based sampling, the APM metrics are computed **on the sampled traffic**, since only the sampled traffic is sent to the OpenTelemetry Collector or Datadog Agent, which is where the metrics calculation is done.
+#### Considerations
 
-To get accurate stats, you can upscale the metrics by using [formulas and functions][7] in Datadog dashboards and monitors, provided that you know the configured sampling rate in the SDK.
+Head-based sampling affects the computation of APM metrics because only the sampled traces are sent to the OpenTelemetry Collector or Datadog Agent, and this is where the metrics calculation is done.
+
+To get accurate metrics, upscale them using [formulas and functions][7] in Datadog dashboards and monitors, provided that you know the configured sampling rate in the SDK.
 
 Use the [ingestion volume control guide][8] to read more about the implications of setting up trace sampling on trace analytics monitors and metrics from spans.
 
-### Collector-level sampling
 
-At the OpenTelemetry collector level, you can do _tail-based sampling_, which allows you to define more advanced rules to keep accrued visibility over error or high latency traces.
+### Tail-based sampling
+
+At the OpenTelemetry Collector level, you can do _tail-based sampling_, which allows you to define more advanced rules to keep accrued visibility over error or high latency traces.
```

Comment:
```suggestion
At the OpenTelemetry Collector level, you can do _tail-based sampling_, which allows you to define more advanced rules to keep accrued visibility over errors or high latency traces.
```

could use a confidence check on whether i'm parsing this sentence correctly... should it actually be "visibility over error traces or high latency traces"?

Reply:
Good thinking. I think we could clarify that with something like:
```suggestion
At the OpenTelemetry Collector level, you can do _tail-based sampling_, which allows you to define more advanced rules to maintain visibility over traces with errors or high latency.
```